python-regression/tests/features/machine2/2_transaction_tests.feature

```diff
@@ -0,0 +1,66 @@
+Feature: Test transaction confirmation
+
+    Scenario: Zero Value Transactions are confirmed
+        In this test, a number of zero value transactions will be made to a specified node.
+        A milestone will be issued that references these transactions, and this should
+        confirm the transations.
+
+        Given "10" transactions are issued on "nodeA" with:
+        |keys                   |values                     |type           |
+        |address                |TEST_ADDRESS               |staticValue    |
+        |value                  |0                          |int            |
+        |tag                    |ZERO9VALUE                 |string         |
+
+        And "findTransactions" is called on "nodeA" with:
+        |keys                   |values                     |type           |
+        |tags                   |ZERO9VALUE                 |list           |
+
+        Then the response for "findTransactions" on "nodeA" is stored in the static value "FOUND_TRANSACTIONS"
+        #In the default DB, the current index is 50. The next milestone issued should be 51.
```

Comment:
The step that this comment was referring to has been removed now and the logic rearranged for pulling the `findTransactions` results for the `getInclusionStates` step. There will be a further breakdown in the API Util logic as well since the list of fetching logic is starting to become erroneous.

Reply:
That `findTransactions` call looks like an unnecessary middle step...

If you wanted to add it as an extra assertion that the transactions were issued it should be explicitly written (and then under the hood it can the current function + another function that does the assertion)

However, If it is here because it is part of the implementation then:
1. You don't need it... you can just use the transactions created in the first step
2. Regardless of 1, implementation steps shouldn't be here

Reply:
I made an adjustment to the transaction logic to forego the need for the `findTransactions` steps. The step was there to provide a storage point for the transactions to be pulled for reference in the milestone step, but the new change makes that unnecessary.